docusaurus.config.js

```diff
@@ -57,37 +57,78 @@ const config = {
     /** @type {import('@docusaurus/preset-classic').ThemeConfig} */
     ({
       navbar: {
-        title: 'Eclipse Tractus-X',
+        title: "Eclipse Tractus-X",
         logo: {
-          alt: 'Eclipse Tractus-X logo',
-          src: 'img/logo.svg',
+          alt: "Eclipse Tractus-X logo",
+          src: "img/logo_catena.svg",
         },
         items: [
+          // {
```

Comment:
Why are these navitems removed? Especially doc is needed, because the site is already actively used for the release guidelines. I cannot find them in the new items array